includes/widgets/video.php

```diff
@@ -1084,6 +1084,10 @@ public function get_embed_params() {
 			$params['color'] = str_replace( '#', '', $settings['color'] );
 
 			$params['autopause'] = '0';
+
+			if ( ! empty( $settings['yt_privacy'] ) ) {
+				$params['dnt'] = 'true';
+			}
```

Comment:
@KingYes Maybe it should be here:
https://github.com/elementor/elementor/blob/master/assets/dev/js/frontend/handlers/video.js#L114